### DISCUSSION TITLE
How to run this example(TestMessage) on a Ubuntu?

### DISCUSSION THREAD

**Discussion by huajuhong** [jmmorato/openddsharp#235] [Q&A]
环境：
Ubuntu 22.04.4 LTS 64-bit

编译：
![image](https://github.com/jmmorato/openddsharp/assets/35752873/6e5b20ce-b345-4f73-9c9f-df46407b889d)

执行命令：
dotnet HelloWorldPublisher.dll

执行结果：
Unhandled exception. System.DllNotFoundException: Unable to load shared library 'TestMessageWrapper' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libTestMessageWrapper: cannot open shared object file: No such file or directory
   at HelloWorld.MessageTypeSupport.MessageTypeSupportNew()
   at HelloWorldPublisher.Program.Main(String[] args) in C:\Users\Administrator\source\repos\OpenDDSSharp\HelloWorldPublisher\Program.cs:line 30
Aborted (core dumped)

**Comment by jmmorato** [comment]
You should build the project directly from an ubuntu machine. Cross-compilation is not working well at the moment

**Reply by huajuhong** [reply]
Okay, thank you. I'll give it a try!
